src/PrestaShopBundle/Entity/Lang.php

```diff
@@ -356,10 +357,20 @@ public function removeShop(Shop $shop)
     /**
      * Get shops.
      *
-     * @return \Doctrine\Common\Collections\Collection
+     * @return Collection
      */
     public function getShops()
     {
         return $this->shops;
     }
+
+    /**
+     * Get translations.
+     *
+     * @return Collection
+     */
+    public function getTranslations()
+    {
+        return $this->translations;
+    }
```

Comment:
Is this called somewhere? I must have didn't see it 😅

Reply:
No, I added this method, else the attribute `$translations` is not used and PHPStan want that I remove it.

Reply:
I think it's the same kind of error that you ignored in `phpstan.neon.dist`:
```yaml
## Doctrine Entities
    - '#Property PrestaShopBundle\\Entity\\[A-Za-z]+\:\:\$[A-Za-z]+ is never written, only read.#'
```

Reply:
I don't think so. What is the interest to have a property if we can access it ? 

The presence of the property is logical. It can be used but it lacks an accessor.